content/administration/install/deploy.rst

```diff
@@ -214,18 +229,7 @@ Needed RAM = #worker * ( (light_worker_ratio * light_worker_ram_estimation) + (h
 LiveChat
 --------
 
-In multiprocessing, a dedicated LiveChat worker is automatically started and
-listening on :option:`the longpolling port <odoo-bin --longpolling-port>` but
-the client will not connect to it.
-
-Instead you must have a proxy redirecting requests whose URL starts with
-``/longpolling/`` to the longpolling port. Other request should be proxied to
-the :option:`normal HTTP port <odoo-bin --http-port>`
-
-To achieve such a thing, you'll need to deploy a reverse proxy in front of Odoo,
-like nginx or apache. When doing so, you'll need to forward some more http Headers
-to Odoo, and activate the proxy_mode in Odoo configuration to have Odoo read those
-headers.
+In multi-processing, a dedicated LiveChat worker is automatically started and listens on the :option:`--longpolling-port <odoo-bin --longpolling-port>`. By default, the HTTP requests will keep accessing the normal HTTP workers instead of the LiveChat one, you must deploy a proxy in front of Odoo and redirect incoming requests whose path starts with ``/longpolling`` to the LiveChat worker. You must also start Odoo in :option:`--proxy-mode <odoo-bin --proxy-mode>` so it uses the real client headers (such as hostname, scheme, and IP) instead of the proxy ones.
```

Comment:
That's a veeery long line :wink:

Reply:
fixed, I also made sure that all other lines where wrap at 100 chars (documentation guidelines)